tests/Move_always_test.cpp

```
@@ -123,6 +145,47 @@ SCENARIO("MoveAlways member functions" * doctest::test_suite("move_always"))
+  CHECK_EQ(strategy.checkpoint_events(), 2);
+  CHECK_EQ(second_attempted, second_succeeded + second_failed);
+
+  MoveAlways_3 replay(passes, checkpoint, seed, false);
+  auto const   replay_first_result = replay(initial);
+  CHECK_EQ(first_result.delaunay_snapshot(),
+           replay_first_result.delaunay_snapshot());
+  CHECK_EQ(first_attempted, replay.get_attempted().total());
+  CHECK_EQ(first_succeeded, replay.get_succeeded().total());
+  CHECK_EQ(first_failed, replay.get_failed().total());
+
+  auto const replay_second_result = replay(initial);
```

Cppcheck (2.21.0)

[performance] Function parameter 'identity' should be passed by const reference.

(passedByValue)